sgpc3_with_shtc1/README.md

```diff
@@ -0,0 +1,9 @@
+# SGPC3 with SHTC1: A Humidity-compensated Combo Driver for SGPC3
+This folder contains a humidity compensated combo-driver combining the SHTC3 and
+the SHTC1 (family compatible like SHTW1, SHTW2, SHTC3) driver into one with the
+necessary bits for the humidity compensation of the gas signal. It is based on
+the SVM30 driver (hence the name.)
```

Comment:
I find that confusing if it is also named SVM30.

Reply:
Fixed